docs/libertyCreate.md

```diff
@@ -55,3 +55,42 @@ liberty {
     }
 }
 ```
+
+4. Override configuration specified above with an `ext` block in `build.gradle`, with a `gradle.properties` file or with project properties specified on the command line. The following server extension properties can be overridden.
+
+* bootstrapProperties
+* defaultVar
+* env
+* jvmOptions
+* var
+
+Those backed by a Properties object can be overridden as a whole or by specifying individual properties. The `jvmOptions` can only be overridden as a whole. When overriding in the `build.gradle` file using an `ext` block, they must be contained in an `afterEvaluate` block so that the `liberty.server` extension exists. 
+
+Examples of using build.gradle file:
+```xml
+<afterEvaluate>
```

Comment:
We don't need to use afterEvaluate anymore. The server extension should exist from the start now.

Also I'd change this from xml to groovy. Should just be 
```
ext {
    liberty.server.env."another.serverenv.var" = "anotherValue"
    liberty.server.defaultVar.someDefaultVar = 'someDefaultValue'
    liberty.server.var.someVar = 'someValue'
    liberty.server.var."my.custom.var" = 'myCustomValue'
    liberty.server.bootstrapProperties."default.http.port" = '9083'
    liberty.server.jvmOptions=['-Xms128m','-Xmx2048m']
}
```